infra/base-images/base-builder/compile

```diff
@@ -186,6 +200,25 @@ else
   fi
 fi
 
+if [ "$SANITIZER" = "introspector" ]; then
```

Comment:
Please merge this with previous condition.

Reply:
I believe there is logic dependency between these two cases placements: the first one prepares env for BUILD_CMD and the second one uses the build command products.
So it cannot be merged into one condition.

Reply:
ACK